.kitchen.yml

```diff
@@ -6,13 +6,15 @@ provisioner:
 
 platforms:
   - name: centos-6.7
-  - name: centos-7.1
-  - name: debian-7.8
+  - name: centos-7.2
+  - name: debian-7.9
     run_list: apt::default
-  - name: debian-8.1
+  - name: debian-8.2
     run_list: apt::default
-  - name: fedora-21
   - name: fedora-22
+    run_list: test::dnf
```

Comment:
We added a DNF backwards compat recipe to the last version of the yum cookbook.  It might make more sense to use that here similar to how we use apt::default